service-api/src/main/java/greencity/dto/event/EventSendEmailDto.java

```diff
@@ -0,0 +1,42 @@
+package greencity.dto.event;
+
+import greencity.dto.user.PlaceAuthorDto;
+import lombok.*;
+
+import java.time.LocalTime;
+import java.util.List;
+
+@Builder
+@Getter
+@Setter
+@ToString
```

Comment:
Why do you need @ToString here?
I mean, what are You going to do with this method?
In my opinion, it is redundant here.